chart/templates/NOTES.txt

```diff
@@ -66,6 +66,16 @@ You can get Fernet Key value by running the following:
 
 {{- end }}
 
+{{- if or (eq .Values.executor "KubernetesExecutor") (eq .Values.executor "CeleryKubernetesExecutor") }}
+{{- if and (not .Values.logs.persistence.enabled) (eq (lower (tpl .Values.config.logging.remote_logging .)) "false") }}
```

Comment:
If a user sets remote logging via Environment variable, this won't work, right?

Reply:
Correct. I considered looking at `env`, but nothing we can really do for `extraEnvFrom` or if its baked into the image so I figured 🤷‍♂️.

Reply:
I also figure a false warning is better than no warning in this situation, but I'm also fine doing nothing if others don't agree.

Reply:
Probably just add a line to say if you are using Env var or secrets backend for setting Airflow configuration ignore this warning